iphone/cli/commands/_build.js

```diff
@@ -2553,11 +2553,17 @@ iOSBuilder.prototype.determineLogServerPort = function determineLogServerPort(ne
 
 			let client = null;
 
-			function die() {
+			function die(error) {
 				client && client.destroy();
-				_t.logger.error(__('Another process is currently bound to port %d', _t.tiLogServerPort));
-				_t.logger.error(__('Set a unique <log-server-port> between 1024 and 65535 in the <ios> section of the tiapp.xml') + '\n');
-				process.exit(1);
+				if (error.code === 'ENOTFOUND') {
+					_t.logger.error(__('Unable to connect to log server on localhost'));
+					_t.logger.error(__('Please ensure your /etc/hosts file contains a valid entry for `localhost`'));
+					process.exit(1);
+				} else {
+					_t.logger.error(__('Another process is currently bound to port %d', _t.tiLogServerPort));
+					_t.logger.error(__('Set a unique <log-server-port> between 1024 and 65535 in the <ios> section of the tiapp.xml') + '\n');
+					process.exit(1);
```

Comment:
No need to duplicate `process.exit(1)`. A single call outside the `if` block would suffice.

Reply:
Updated